docker-compose.yml

```diff
@@ -0,0 +1,11 @@
+web:
+  build: .
+  command: mix phoenix.server
+  environment:
+    PORT: 80
+  volumes:
+  - .:/app
+  ports:
+  - "80:80"
```

Comment:
I've always got confused with this part of using docker/container, to me it feels like, when I set up a container I shouldn't think about other independent systems, is this right? Basically APR shouldn't know about Gravity.